catalogue/webapp/pages/api/works/[workId].tsx

```diff
@@ -33,6 +33,7 @@ const WorksApi = async (
 
   res.setHeader('Content-Type', 'application/json');
   res.setHeader('Access-Control-Allow-Origin', '*');
+  // we are forcing revalidation at all times because this is to do with ite availability, so it must be up to date
```

Comment:
Don't think this is true here(?), this is just a normal catalogue API work - fine to cache I think

Reply:
done